Restructure tabs

laurensWe opened the pull request:
# Description

- Bring widgets with requirements and tools to the algorithms details tab
- Rename requirements tab to compliance
- Delete model & data tabs
- Delete tools tab
- Make a placeholder member tab
- Edit the unit tests

Resolves #388 

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x] I have tested the changes locally and verified that they work as expected.
- [x] I have followed the project's coding conventions and style guidelines.
- [x] I have rebased my branch onto the latest commit of the main branch.
- [x] I have squashed or reorganized my commits into logical units.
- [x] I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

ravimeijerrig reviewed (approved):
LGTM

ravimeijerrig commented:
@robbertbos we think of the following adjustments 
- Leden tab can be removed for the pilot phase 
- Link to all requirements must be checked from widget if it works properly and redirects to the 'Voldoen aan wetten en regels' tab. 
- Remove Instrument widget (make measure widget full width), because it is not yet implemented.

laurensWe commented:
Whether the 'Voldoen aan wetten en regels' tab is correct is whether the algorithms have been deleted and remade, because now the datamodel of a measure also has "accountable_persons", "reviewer_persons", and "responsible_persons" in it.

robbertbos commented:
@ravimeijerrig my thoughts:

1. Agree to remove "People" tab for pilot phase; we will add it once we implemented the permissions.
2. Didn't we decide to use a shorter name for the tabs? I thought we decided to keep it short as-in "Requirements". On the tab section we can keep a longer title.
3. Agree that we can remove it for now. We can list the selected instruments on the algorithm details section. And once implemented, we can add a new tab where people can work on an instrument.

laurensWe commented:
> @ravimeijerrig my thoughts:
> 
>     1. Agree to remove "People" tab for pilot phase; we will add it once we implemented the permissions.
> 
>     2. Didn't we decide to use a shorter name for the tabs? I thought we decided to keep it short as-in "Requirements". On the tab section we can keep a longer title.
> 
>     3. Agree that we can remove it for now. We can list the selected instruments on the algorithm details section. And once implemented, we can add a new tab where people can work on an instrument.

1. and 3. are now implemented, I am waiting both your replies on the second item what to change there

ravimeijerrig commented:
@robbertbos @laurensWe

With this comment we meant that the link in the last line of the widget needs to be checked. I see that in production this link is working again.

To clarify the naming: 
This is already the short name (compared to 'Voldoen aan wet- en regelgeving') and it aligns with AK (see design in [ticket](https://github.com/orgs/MinBZK/projects/7/views/3?pane=issue&itemId=86472743&issue=MinBZK%7Camt%7C388)). What is the difference between 'tabs' and 'tabs section'?

robbertbos commented:
Let's keep the current proposal "as-is" and follow what's in #388 (@laurensWe). When we add more tabs, we can always change it ;) 

@ravimeijerrig to clarify:

- Tabs: the "row" with tabs
- Tab section: the content of the tab when you click on it

ravimeijerrig reviewed (approved):
LGTM, if the term 'Compliance' as translation from 'Voldoen aan wetten en regels' is agreed

uittenbroekrobbert reviewed (approved):
LGTM!
